iothub/service/src/Registry/Models/Job.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Contains the properties set by the service import/export job.
+    /// </summary>
+    public abstract class Job
```

Comment:
Job is too simple of a name. We could say IotHubJobProperties or IotHubJobResult or IotHubJobResponse. I probably prefer the last name most.